packages/cdktf/lib/terraform-resource.ts

```diff
@@ -127,6 +127,18 @@ export class TerraformResource
     };
   }
 
+  public toMetadata(): any {
+    if (!Object.keys(this.rawOverrides).length) {
+      return {};
+    }
+
+    return {
+      overrides: {
+        [this.terraformResourceType]: Object.keys(this.rawOverrides),
```

Comment:
Overrides exist on all `TerraformElements`; why only record for resources?

Reply:
Yeah good point, I'll add them across the board!